datalad_crawler/nodes/annex.py

```diff
@@ -1320,7 +1320,11 @@ def _finalize(data):
 
             if aggregate:
                 from datalad.api import aggregate_metadata
-                aggregate_metadata(dataset='^', path=self.repo.path, update_mode='all')
+                aggregate_metadata(
+                    dataset='^',
+                    path=self.repo.path,
+                    update_mode='all',
+                    incremental=True)
```

Comment:
@mih, what is currently a use case for having all "neighboring" subdatasets in superdatasets to loose their metadata whenever I aggregate without `incremental=True`?

Reply:
@mih -- with `incremental` which I have to use so other subdatasets do not loose their metadata, now tests fail (e.g. https://travis-ci.org/datalad/datalad-crawler/jobs/410135254) .
 I think things need fix ups on datalad side so https://github.com/datalad/datalad/issues/2772 . Meanwhile I will fixup here by assuming that may be pruning of old objects would come in the same commit as updating them, so count would decrease ;-)